### PR TITLE
Refactor/category levels

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: webapp/components/survey/CategoryDetails/LevelDetails.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
```
Comment (minotogna) [inline webapp/components/survey/CategoryDetails/LevelDetails.js:3]: this should go to its own folder.

File: webapp/components/survey/CategoryDetails/store/actions/useFetchLevelItems.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const surveyId = useSurveyId()
+
+  return useCallback(async ({ categoryUuid, levelIndex = 0, parentUuid = null }) => {
+    // Reset level items
+    setState(State.dissocLevelItems({ levelIndex }))
+
+    const {
+      data: { items },
+    } = await axios.get(`/api/survey/${surveyId}/categories/${categoryUuid}/items`, {
+      params: { draft: true, parentUuid },
```
Comment (minotogna) [inline webapp/components/survey/CategoryDetails/store/actions/useFetchLevelItems.js:18]: isn't draft = canEdit ?

Comment (SteRiccio) [reply webapp/components/survey/CategoryDetails/store/actions/useFetchLevelItems.js:18]: right!

File: webapp/components/survey/CategoryDetails/LevelDetails.js (hunk 1 above)
Comment (SteRiccio) [reply webapp/components/survey/CategoryDetails/LevelDetails.js:3]: ok

**Pull request merged**